texture-font.c

```diff
@@ -621,7 +621,7 @@ texture_font_load_glyph( texture_font_t * self,
     unsigned char *src_ptr = ft_bitmap.buffer;
     for( i = 0; i < src_h; i++ )
     {
-        memcpy( dst_ptr, src_ptr, ft_bitmap.pitch);
+        memcpy( dst_ptr, src_ptr, ft_bitmap.width);
```

Comment:
this should suffice 
https://www.freetype.org/freetype2/docs/reference/ft2-basic_types.html#FT_Bitmap